server/src/main/java/org/opensearch/index/store/remote/filecache/FileCache.java

```diff
@@ -49,8 +50,12 @@ public class FileCache implements RefCountedCache<Path, CachedIndexInput> {
 
     private final CircuitBreaker circuitBreaker;
 
-    // TODO: Convert the constant into an integer setting
-    public static final int DATA_TO_FILE_CACHE_SIZE_RATIO = 5;
+    public static final Setting<Integer> DATA_TO_FILE_CACHE_SIZE_RATIO_SETTING = Setting.intSetting(
```

Comment:
Should this be double ?

Reply:
And added to `ClusterSettings` since it look like one? Should it be dynamic (`Property.Dynamic`) to not require nodes restart?

Reply:
Updated to `double`, Dynamic and added an entry into ClusterSettings. 
Thanks for the catch @Bukhtawar and @reta!